pkgs/desktops/pantheon/apps/elementary-mail/default.nix

```diff
@@ -0,0 +1,82 @@
+{ lib, stdenv
+, fetchFromGitHub
+, nix-update-script
+, pantheon
+, pkg-config
+, meson
+, ninja
+, python3
+, vala
+, desktop-file-utils
+, gtk3
+, libxml2
+, libhandy
+, webkitgtk
+, folks
+, libgdata
+, sqlite
+, granite
+, elementary-icon-theme
+, evolution-data-server
+, appstream
+, wrapGAppsHook
+, libgee
+}:
+
+stdenv.mkDerivation rec {
+  pname = "elementary-mail";
+  version = "unstable-2021-06-21";
+
+  repoName = "mail";
+
+  src = fetchFromGitHub {
+    owner = "elementary";
+    repo = repoName;
```

Comment:
```suggestion
  src = fetchFromGitHub {
    owner = "elementary";
    repo = "mail";
```

Reply:
All the other files do this, I did it this way to keep it uniform.

Reply:
See:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/desktops/pantheon/apps/elementary-photos/default.nix#L40-L46
https://github.com/NixOS/nixpkgs/blob/master/pkgs/desktops/pantheon/apps/elementary-photos/default.nix#L40-L47
Sure its a bit weird, and perhaps unnecessary, but I think all the applications should stay uniform.